src/gatsby/src/pages/plan-your-trip/park-operating-dates.js

```diff
@@ -54,6 +59,42 @@ const ParkLink = ({ park, advisories }) => {
     }
   }
 
+  // get advisories
+  const getAccessStatus = (status) => {
+    setParkAccessStatus(status)
+  }
+
+  useEffect(() => {
+    setIsLoadingAdvisories(true)
+    loadAdvisories(apiBaseUrl, park.orcs)
+      .then(response => {
+        if (response.status === 200) {
+          setAdvisories(response.data.data)
+          setAdvisoryLoadError(false)
+        } else {
+          setAdvisories([])
+          setAdvisoryLoadError(true)
+        }
```

Comment:
If the API sends back an error code like 404 or 500, the axios call in `loadAdvisories` will throw an exception, so those errors would go to a `.catch` block instead of the `.then`, I think!